JankStatsSample/app/src/main/java/com/example/jankstats/compose/ComposeListFragment.kt

```diff
@@ -71,6 +71,7 @@ class ComposeListFragment : Fragment() {
 @Composable
 fun MessageList(onItemClick: () -> Unit) {
     val listState = rememberLazyListState()
+    // [START compose_jank_metrics]
     val metricsStateHolder = rememberMetricsStateHolder()
```

Comment:
can we add it to the `rememberMetricsStateHolder` as well?

Reply:
Added